api/app/db/models/thessian_polygon_areas.py

```diff
@@ -0,0 +1,19 @@
+""" Class models that reflect resources and map to database tables for HFI Calculator.
+"""
+from geoalchemy2.types import Geometry
+from sqlalchemy import (Column, Integer,
+                        Sequence)
+from app.db.database import Base
+
+
+class ThessianPolygonArea(Base):
+    """ Thessian polygon for a given area """
```

Comment:
We need to comply to standards, so we need to add in a "create_date" and "update_date"
e.g.:

`create_date = Column(TZTimeStamp, nullable=False,
                         default=time_utils.get_utc_now())
    # Date this record was updated.
    update_date = Column(TZTimeStamp, nullable=False,
                         default=time_utils.get_utc_now())`

Reply:
Hmm, we don't have these on all tables, and the tables that do have them have different field names so I didn't think there was a standard.